src/Farmer/Arm/ResourceGroup.fs

```diff
@@ -32,19 +33,23 @@ type ResourceGroupDeployment =
                 (function 
                 | :? ResourceGroupDeployment as rg -> rg.RequiredResourceGroups 
                 | _ ->  [])
-        List.distinct (this.Name.Value :: nestedRgs)        
+        if this.ResourceName.[0] = '[' then
```

Comment:
There probably is but not sure what it could be (beginner), I know that when we don't do this it appends the resource group name that we added in the webapp builder (`"[resourceGroup.Name()]"`) and we end up with a console log llike this:

```
Creating resource group my-resource-group-name-2 (1/2)...
Creating resource group [resourceGroup().name] (2/2)...
```